The job **`x86_64-gnu-llvm-17`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/10750060875/job/29815752748) [(plain)](https://github.com/rust-lang/rust/commit/8230a90c494dcc7d8013a0a15719cc9f2f9a8263/checks/29815752748/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
------
 > importing cache manifest from ghcr.io/rust-lang/rust-ci-cache:fb97a1d6377f6cf2227825318ca4bbde3889e0c420746f5a03ba46a07e9a862725c26a09b9fc49a0d129ebd75935d3f6cd19acf41cc4267a6846fd4aa574b12c:
------
##[endgroup]
Setting extra environment values for docker:  --env ENABLE_GCC_CODEGEN=1 --env GCC_EXEC_PREFIX=/usr/lib/gcc/
[CI_JOB_NAME=x86_64-gnu-llvm-17]
---
sccache: Starting the server...
##[group]Configure the build
configure: processing command line
configure: 
configure: build.configure-args := ['--build=x86_64-unknown-linux-gnu', '--llvm-root=/usr/lib/llvm-17', '--enable-llvm-link-shared', '--set', 'rust.randomize-layout=true', '--set', 'rust.thin-lto-import-instr-limit=10', '--set', 'change-id=99999999', '--enable-verbose-configure', '--enable-sccache', '--disable-manage-submodules', '--enable-locked-deps', '--enable-cargo-native-static', '--set', 'rust.codegen-units-std=1', '--set', 'dist.compression-profile=balanced', '--dist-compression-formats=xz', '--set', 'rust.lld=false', '--disable-dist-src', '--release-channel=nightly', '--enable-debug-assertions', '--enable-overflow-checks', '--enable-llvm-assertions', '--set', 'rust.verify-llvm-ir', '--set', 'rust.codegen-backends=llvm,cranelift,gcc', '--set', 'llvm.static-libstdcpp', '--enable-new-symbol-mangling']
configure: target.x86_64-unknown-linux-gnu.llvm-config := /usr/lib/llvm-17/bin/llvm-config
configure: llvm.link-shared     := True
configure: rust.randomize-layout := True
configure: rust.thin-lto-import-instr-limit := 10
---
---- [ui] tests/ui/privacy/suggest-box-new.rs stdout ----
diff of stderr:

44    |                      ~~~~~~~~~~~~~~
45 LL |         wtf: Some(Box::new_in(_, _)),
-      and 10 other candidates
+      and 12 other candidates
48 help: consider using the `Default` trait
49    |
49    |
50 LL |         wtf: Some(<Box as std::default::Default>::default()),
89    |                ~~~~~~~~~~~~~~
90 LL |     let _ = Box::new_in(_, _);
91    |                ~~~~~~~~~~~~~~
-      and 10 other candidates
---
To only update this specific test, also pass `--test-args privacy/suggest-box-new.rs`

error: 1 errors occurred comparing output.
status: exit status: 1
command: env -u RUSTC_LOG_COLOR RUSTC_ICE="0" RUST_BACKTRACE="short" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc" "/checkout/tests/ui/privacy/suggest-box-new.rs" "-Zthreads=1" "-Zsimulate-remapped-rust-src-base=/rustc/FAKE_PREFIX" "-Ztranslate-remapped-path-to-local-path=no" "-Z" "ignore-directory-in-diagnostics-source-blocks=/cargo" "-Z" "ignore-directory-in-diagnostics-source-blocks=/checkout/vendor" "--sysroot" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage2" "--target=x86_64-unknown-linux-gnu" "--check-cfg" "cfg(FALSE)" "--error-format" "json" "--json" "future-incompat" "-Ccodegen-units=1" "-Zui-testing" "-Zdeduplicate-diagnostics=no" "-Zwrite-long-types-to-disk=no" "-Cstrip=debuginfo" "--emit" "metadata" "-C" "prefer-dynamic" "--out-dir" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/ui/privacy/suggest-box-new" "-A" "unused" "-A" "internal_features" "-Crpath" "-Cdebuginfo=0" "-Lnative=/checkout/obj/build/x86_64-unknown-linux-gnu/native/rust-test-helpers" "-L" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/ui/privacy/suggest-box-new/auxiliary"
--- stderr -------------------------------
error[E0423]: expected function, tuple struct or tuple variant, found struct `std::collections::HashMap`
##[error]  --> /checkout/tests/ui/privacy/suggest-box-new.rs:14:13
   |
---
LL |     let _ = std::collections::HashMap::with_capacity(_);
   |                                      ~~~~~~~~~~~~~~~~~~
LL |     let _ = std::collections::HashMap::with_hasher(_);
   |                                      ~~~~~~~~~~~~~~~~
LL |     let _ = std::collections::HashMap::with_capacity_and_hasher(_, _);
help: consider using the `Default` trait
   |
LL |     let _ = <std::collections::HashMap as std::default::Default>::default();
   |             +                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   |             +                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error[E0423]: cannot initialize a tuple struct which contains private fields
##[error]  --> /checkout/tests/ui/privacy/suggest-box-new.rs:8:19
   |
LL |         wtf: Some(Box(U { //~ ERROR cannot initialize a tuple struct which contains private fields
   |
note: constructor is not visible here due to private fields
  --> /rustc/FAKE_PREFIX/library/alloc/src/boxed.rs:236:14
   |
   |
   = note: private field
   |
   = note: private field
help: you might have meant to use an associated function to build this type
   |
LL |         wtf: Some(Box::new(_)),
   |                      ~~~~~~~~
LL |         wtf: Some(Box::new_uninit()),
   |                      ~~~~~~~~~~~~~~
LL |         wtf: Some(Box::new_zeroed()),
   |                      ~~~~~~~~~~~~~~
LL |         wtf: Some(Box::new_in(_, _)),
     and 12 other candidates
help: consider using the `Default` trait
   |
   |
LL |         wtf: Some(<Box as std::default::Default>::default()),

error: cannot construct `HashMap<_, _, _>` with struct literal syntax due to private fields
##[error]  --> /checkout/tests/ui/privacy/suggest-box-new.rs:16:13
   |
---
LL |     let _ = std::collections::HashMap::with_capacity(_);
   |                                      ~~~~~~~~~~~~~~~~~~
LL |     let _ = std::collections::HashMap::with_hasher(_);
   |                                      ~~~~~~~~~~~~~~~~
LL |     let _ = std::collections::HashMap::with_capacity_and_hasher(_, _);
help: consider using the `Default` trait
   |
LL |     let _ = <std::collections::HashMap as std::default::Default>::default();
   |             +                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   |             +                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error: cannot construct `Box<_, _>` with struct literal syntax due to private fields
##[error]  --> /checkout/tests/ui/privacy/suggest-box-new.rs:18:13
   |
LL |     let _ = Box {}; //~ ERROR cannot construct `Box<_, _>` with struct literal syntax due to private fields
   |
   = note: private fields `0` and `1` that were not provided
help: you might have meant to use an associated function to build this type
   |
   |
LL |     let _ = Box::new(_); //~ ERROR cannot construct `Box<_, _>` with struct literal syntax due to private fields
   |                ~~~~~~~~
LL |     let _ = Box::new_uninit(); //~ ERROR cannot construct `Box<_, _>` with struct literal syntax due to private fields
   |                ~~~~~~~~~~~~~~
LL |     let _ = Box::new_zeroed(); //~ ERROR cannot construct `Box<_, _>` with struct literal syntax due to private fields
   |                ~~~~~~~~~~~~~~
LL |     let _ = Box::new_in(_, _); //~ ERROR cannot construct `Box<_, _>` with struct literal syntax due to private fields
     and 12 other candidates
help: consider using the `Default` trait
   |
   |
LL |     let _ = <Box as std::default::Default>::default(); //~ ERROR cannot construct `Box<_, _>` with struct literal syntax due to private fields

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0423`.
---
To only update this specific test, also pass `--test-args suggestions/deref-path-method.rs`

error: 1 errors occurred comparing output.
status: exit status: 1
command: env -u RUSTC_LOG_COLOR RUSTC_ICE="0" RUST_BACKTRACE="short" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc" "/checkout/tests/ui/suggestions/deref-path-method.rs" "-Zthreads=1" "-Zsimulate-remapped-rust-src-base=/rustc/FAKE_PREFIX" "-Ztranslate-remapped-path-to-local-path=no" "-Z" "ignore-directory-in-diagnostics-source-blocks=/cargo" "-Z" "ignore-directory-in-diagnostics-source-blocks=/checkout/vendor" "--sysroot" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage2" "--target=x86_64-unknown-linux-gnu" "--check-cfg" "cfg(FALSE)" "--error-format" "json" "--json" "future-incompat" "-Ccodegen-units=1" "-Zui-testing" "-Zdeduplicate-diagnostics=no" "-Zwrite-long-types-to-disk=no" "-Cstrip=debuginfo" "--emit" "metadata" "-C" "prefer-dynamic" "--out-dir" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/ui/suggestions/deref-path-method" "-A" "unused" "-A" "internal_features" "-Crpath" "-Cdebuginfo=0" "-Lnative=/checkout/obj/build/x86_64-unknown-linux-gnu/native/rust-test-helpers" "-L" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/ui/suggestions/deref-path-method/auxiliary"
--- stderr -------------------------------
error[E0599]: no function or associated item named `contains` found for struct `Vec<_, _>` in the current scope
##[error]  --> /checkout/tests/ui/suggestions/deref-path-method.rs:3:10
   |
   |
LL |     Vec::contains(&vec, &0);
   |          ^^^^^^^^ function or associated item not found in `Vec<_, _>`
   |
note: if you're trying to build a new `Vec<_, _>` consider using one of the following associated functions:
      Vec::<T>::new
      Vec::<T>::with_capacity
      Vec::<T>::try_with_capacity
      Vec::<T>::from_raw_parts
  --> /rustc/FAKE_PREFIX/library/alloc/src/vec/mod.rs:421:5
help: the function `contains` is implemented on `[_]`
   |
   |
LL |     <[_]>::contains(&vec, &0);

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0599`.
---
To only update this specific test, also pass `--test-args ufcs/bad-builder.rs`

error: 1 errors occurred comparing output.
status: exit status: 1
command: env -u RUSTC_LOG_COLOR RUSTC_ICE="0" RUST_BACKTRACE="short" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc" "/checkout/tests/ui/ufcs/bad-builder.rs" "-Zthreads=1" "-Zsimulate-remapped-rust-src-base=/rustc/FAKE_PREFIX" "-Ztranslate-remapped-path-to-local-path=no" "-Z" "ignore-directory-in-diagnostics-source-blocks=/cargo" "-Z" "ignore-directory-in-diagnostics-source-blocks=/checkout/vendor" "--sysroot" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage2" "--target=x86_64-unknown-linux-gnu" "--check-cfg" "cfg(FALSE)" "--error-format" "json" "--json" "future-incompat" "-Ccodegen-units=1" "-Zui-testing" "-Zdeduplicate-diagnostics=no" "-Zwrite-long-types-to-disk=no" "-Cstrip=debuginfo" "--emit" "metadata" "-C" "prefer-dynamic" "--out-dir" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/ui/ufcs/bad-builder" "-A" "unused" "-A" "internal_features" "-Crpath" "-Cdebuginfo=0" "-Lnative=/checkout/obj/build/x86_64-unknown-linux-gnu/native/rust-test-helpers" "-L" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/ui/ufcs/bad-builder/auxiliary"
--- stderr -------------------------------
--- stderr -------------------------------
error[E0599]: no function or associated item named `mew` found for struct `Vec<Q>` in the current scope
   |
   |
LL |     Vec::<Q>::mew()
   |               ^^^ function or associated item not found in `Vec<Q>`
   |
note: if you're trying to build a new `Vec<Q>` consider using one of the following associated functions:
      Vec::<T>::new
      Vec::<T>::with_capacity
      Vec::<T>::try_with_capacity
      Vec::<T>::from_raw_parts
  --> /rustc/FAKE_PREFIX/library/alloc/src/vec/mod.rs:421:5
help: there is an associated function `new` with a similar name
   |
   |
LL |     Vec::<Q>::new()

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0599`.
```

</details>

--- a/tests/ui/privacy/suggest-box-new.stderr
+++ b/tests/ui/privacy/suggest-box-new.stderr
@@ -44,7 +44,7 @@ LL |         wtf: Some(Box::new_zeroed()),
    |                      ~~~~~~~~~~~~~~
 LL |         wtf: Some(Box::new_in(_, _)),
    |                      ~~~~~~~~~~~~~~
-     and 10 other candidates
+     and 12 other candidates
 help: consider using the `Default` trait
    |
 LL |         wtf: Some(<Box as std::default::Default>::default()),
@@ -89,7 +89,7 @@ LL |     let _ = Box::new_zeroed();
    |                ~~~~~~~~~~~~~~
 LL |     let _ = Box::new_in(_, _);
    |                ~~~~~~~~~~~~~~
-     and 10 other candidates
+     and 12 other candidates
 help: consider using the `Default` trait
    |
 LL |     let _ = <Box as std::default::Default>::default();

--- a/tests/ui/suggestions/deref-path-method.stderr
+++ b/tests/ui/suggestions/deref-path-method.stderr
@@ -9,7 +9,7 @@ note: if you're trying to build a new `Vec<_, _>` consider using one of the foll
       Vec::<T>::with_capacity
       Vec::<T>::try_with_capacity
       Vec::<T>::from_raw_parts
-      and 4 others
+      and 6 others
   --> $SRC_DIR/alloc/src/vec/mod.rs:LL:COL
 help: the function `contains` is implemented on `[_]`
    |

--- a/tests/ui/ufcs/bad-builder.stderr
+++ b/tests/ui/ufcs/bad-builder.stderr
@@ -9,7 +9,7 @@ note: if you're trying to build a new `Vec<Q>` consider using one of the followi
       Vec::<T>::with_capacity
       Vec::<T>::try_with_capacity
       Vec::<T>::from_raw_parts
-      and 4 others
+      and 6 others
   --> $SRC_DIR/alloc/src/vec/mod.rs:LL:COL
 help: there is an associated function `new` with a similar name
    |

